### ISSUE TITLE
error message when MATHHUB variable is not set. 

### ISSUE THREAD

**Issue by kohlhase** [slatex/lmhtools#28]
It seems that this is not reliably done.

**Comment by kohlhase** [comment]
I thought @Jazzpirate said that this was done, but I just had fun with @jfschaefer debugging the problem (again)

**Comment by Jazzpirate** [comment]
This is definitely implemented on master, see https://github.com/slatex/sTeX/blob/master/sty/mathhub/mathhub.dtx#L411

-- however, the error is only thrown in the mathhub package, so `mh` package option needs to be set. I'm guessing other packages *without* `mh` should not depend on the environment variable being set?

**Comment by kohlhase** [comment]
hmmm, I am not sure that I can reconstruct the situation. @jfschaefer can you? it should still be there, ifyou do a `unset MAHTHUB`

**Comment by jfschaefer** [comment]
I've placed the following code as `test.tex` in `MiKoProblems/AI/source/search/en`:
```latex
\documentclass[notes,mh]{mikoslides}
\libinput{preamble}
\begin{document}
\begin{frame}
  \usemhmodule[mhrepos=MiKoMH/AI,dir=search/en]{id-search}
\end{frame}
\end{document}
```

Without `MATHHUB` being set, it results in a long and confusing error message.

**Comment by Jazzpirate** [comment]
Fixed in db70422d68885f1cef14b6ed8191fa99a64bc252

**Comment by kohlhase** [comment]
sorry I just realized that the issue was in the wrong place. thanks for fixing it anyhow.
